scripts/utils.js

```diff
@@ -0,0 +1,35 @@
+const { execSync } = require('child_process');
+const readline = require('readline');
+const { resolve } = require('path');
+
+function getLatestVersion() {
+  return execSync(`npm view @mongosh/cli-repl .dist-tags.latest`).toString().trim();
+}
+
+function gitClone(repo, dest) {
+  return execSync(`git clone ${repo} ${dest}`);
```

Comment:
I know it’s only taken from the code above, but to avoid edge cases with special characters in the expansions:

```suggestion
  return execFileSync('git', ['clone', repo, dest]`);
```

? (This also applies to other places, but I generally consider `execSync()` with template strings/string concatenation something that should be avoided unless the range of input values is fixed somehow)

Reply:
Should be fine given we have full control over those params, however i don't see why we shouldn't improve it, we can change that together.